### PR TITLE
add verification submission comment

### PR DESCRIPTION
**Pull request by ayrat555**
resolves https://github.com/poanetwork/blockscout/issues/2575

## Motivation

img

## Changelog

- add verification submission comment

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Current
 
 ### Features
+- [#1588](https://github.com/poanetwork/blockscout/pull/2588) - add verification submission comment
```
Comment (pasqu4le) [inline CHANGELOG.md]: PR number is `#1588` instead of `#2588`

Comment (ayrat555) [reply CHANGELOG.md]: thanks

**Comment by ayrat555** [comment]
@vbaranov we don't save the comment to the DB so the contract will have only one comment

**Comment by vbaranov** [comment]
@ayrat555 user can copy source code from another verified contract in Blockscout. In this case there will be doubled submission comments. Isn’t it?

**Comment by ayrat555** [comment]
@vbaranov  I added handling of all 4 cases
https://github.com/poanetwork/blockscout/pull/2588/files#diff-1bc9df57fc4a868238834dcfa8cc3520R292

**Comment by ayrat555** [comment]
@vbaranov I fixed it

**Comment by vbaranov** [comment]
>@vbaranov I fixed it

@ayrat555 it still causes internal server error. You can check with the address I previously posted. Please check logs of stg application.

**Comment by ayrat555** [comment]
@vbaranov It loads successfully with `staging-mainnet-stg` branch 
img

**Pull request merged**